portsyncd/portsyncd.cpp

```diff
@@ -250,6 +251,19 @@ void handlePortConfig(ProducerStateTable &p, map<string, KeyOpFieldsValuesTuple>
             if (op == SET_COMMAND)
             {
                 p.set(key, values);
+                for (auto fv : values)
+                {
+                    string field = fvField(fv);
+                    string value = fvValue(fv);
+
+                    /* Update the mtu field on host interface */
+                    if (field == "mtu")
+                    {
+                        string cmd, res;
+                        cmd = "ip link set " + key + " mtu " + value;
+                        swss::exec(cmd, res);
```

Comment:
Do you want to check for return value?.

Reply:
Hi Sunny @prsunny ,

The return value will be checked inside swss::exec, if failed swss::exec will throw an error. 

Thanks,
Haiyang